Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -35,13 +35,14 @@ public async Task<List<CustomCommand>> GetCustomCommands()
         public async Task InitDefaultCustomCommands()
         {
             var customCommands = new List<AddCustomCommandsCommand>();
-            foreach (var serverId in new ulong[] { 636238466899902504, 597066406521208852 })
+            foreach (var serverId in new ulong[] { 636238466899902504, 597066406521208852, 729754533311283293 })
             {
                 customCommands.AddRange(new List<AddCustomCommandsCommand>
                 {
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<Roles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<Roles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Messaging.BotCommands.SendCommand",@"-send\s*(?<Channel>\<.*\>)\s*""?(?<Message>.*)\""?" , serverId),
```

Comment:
zwróć uwagę na spacje w tej linijce

Reply:
nie widziałem tego od razu i analizowałem regexa

` , serverId),`
chodzi o to że przed przecinkiem jest spacja

Reply:
i przed regexem nie ma spacji